src/bin/freq/main.rs

```diff
@@ -37,7 +37,8 @@ fn run_main() -> Result<i32> {
     if let Some(c) = Config::load_from_file(&opts.config_file)? {
         opts.config.merge(c)
     }
-    let cfg = &opts.config;
+    let inputs = opts.inputs();
```

Comment:
I did this and line 41 so I can take ownership of cfg without having partial move problems.